### ISSUE TITLE
What operating environment support does this project require?

### ISSUE THREAD

**Issue by joylix** [giacomociti/rdf2dot#11]
The following error message is prompted when running index.html:
1)  rdf_flyer.24 not found.      (it's in ./public/)
2) index.html:1 Uncaught TypeError: Failed to resolve module specifier "file-saver". Relative references must start with either "/", "./", or "../".
So I didn't get any output.
I tried the online demo page but didn't get the correct output response.

**Comment by giacomociti** [comment]
`index.html` is not meant to be served as is. The project is built with [vitejs](https://vitejs.dev/).
In order to run it locally, you need [nodejs ](https://nodejs.org/) installed and then you can issue the command:

`
npm run dev
`.

The first time you also have to run:

`
npm install
`

**Comment by joylix** [comment]
Do I need to install graphviz?

**Comment by giacomociti** [comment]
no, it's already provided by [@viz-js/viz](https://www.npmjs.com/package/@viz-js/viz)
